### PR TITLE
#187 first version of user documentation

### LINKED ISSUE #187: Create user documentation

### PR DESCRIPTION
**Pull request by rosiereflo**
closes #187

### CODE REVIEW

**Comment by gautierbureau** [comment]
.pdf .log .aux should be removed from the PR

**Comment by gautierbureau** [comment]
You should add a build-doc and doc commands in envDynawoAlgorithms.sh

**Comment by rosiereflo** [comment]
done for both remarks.

File: documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+\end{lstlisting}
+
+The default number of thread is 1. The input can be explicit files or zipped ones. For example:
+
+\begin{lstlisting}[language=bash, breaklines=true, breakatwhitespace=false]
+$> ./myEnvDynawoAlgorithms.sh SA --directory nrt/data/IEEE14/SA --input fic_MULTIPLE.xml --output results.xml --nbThreads 2
+\end{lstlisting}
+
+This command will launch the systematic analysis described in the file \textit{nrt/data/IEEE14/SA/fic\_MULTIPLE.xml} in multi-threading 
+and provide as outputs a file named \textit{results.xml} containing the status of each simulated event,
```
Comment (adriguir) [inline documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex:113]: provides

File: documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+\end{lstlisting}
+
+The default number of thread is 1. The input can be explicit files or zipped ones. For example:
+
+\begin{lstlisting}[language=bash, breaklines=true, breakatwhitespace=false]
+$> ./myEnvDynawoAlgorithms.sh MC --directory nrt/data/IEEE14/MC --input fic_MULTIPLE.xml --output results.xml --nbThreads 2
+\end{lstlisting}
+
+This command will launch the margin calculation described in the file \textit{nrt/data/IEEE14/MC/fic\_MULTIPLE.xml} in multi-threading 
+and provide as outputs a file named \textit{results.xml} containing the status of all simulations that were run,
```
Comment (adriguir) [inline documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex:204]: provides

File: documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+
+The default number of thread is 1. The input can be explicit files or zipped ones. For example:
+
+\begin{lstlisting}[language=bash, breaklines=true, breakatwhitespace=false]
+$> ./myEnvDynawoAlgorithms.sh MC --directory nrt/data/IEEE14/MC --input fic_MULTIPLE.xml --output results.xml --nbThreads 2
+\end{lstlisting}
+
+This command will launch the margin calculation described in the file \textit{nrt/data/IEEE14/MC/fic\_MULTIPLE.xml} in multi-threading 
+and provide as outputs a file named \textit{results.xml} containing the status of all simulations that were run, 
+in addition to the timelines and constraint files.\\
```
Comment (adriguir) [inline documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex:205]: logs too

File: documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+\begin{lstlisting}[language=bash, breaklines=true, breakatwhitespace=false]
+$> ./myEnvDynawoAlgorithms.sh MC --directory nrt/data/IEEE14/MC --input inputs.zip --output outputs.zip
+\end{lstlisting}
+
+This command will unzip the files contained in \textit{inputs.zip}, launch a MC assuming the input file name is \textit{fic\_MULTIPLE.xml}
+and provide as outputs a zipped file named \textit{outputs.zip} containing the status of all simulations that were run,
```
Comment (adriguir) [inline documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex:212]: provides

File: documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+and provide as outputs a file named \textit{results.xml} containing the status of all simulations that were run, 
+in addition to the timelines and constraint files.\\
+
+\begin{lstlisting}[language=bash, breaklines=true, breakatwhitespace=false]
+$> ./myEnvDynawoAlgorithms.sh MC --directory nrt/data/IEEE14/MC --input inputs.zip --output outputs.zip
+\end{lstlisting}
+
+This command will unzip the files contained in \textit{inputs.zip}, launch a MC assuming the input file name is \textit{fic\_MULTIPLE.xml}
+and provide as outputs a zipped file named \textit{outputs.zip} containing the status of all simulations that were run, 
+in addition to the timelines and constraint files.\\
```
Comment (adriguir) [inline documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex:213]: logs too

File: documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+
+\subsubsection{SA mode}
+
+the output files of the SA mode are the timelines and constraint files for all simulations (if they were requested in the base \textbf{.jobs} file), in addition to an aggregated 
+results file with one result per scenario.
+
+\subsubsection{MC mode}
+
+the output files of the SA mode are the timelines and constraint files for all simulations (if they were requested in the base \textbf{.jobs} files), in addition to an aggregated 
+results file with one result per load increase and one result per scenario for each variation that were simulated.
```
Comment (adriguir) [inline documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex:266]: logs again for SA and MC?

File: documentation/introduction/introduction.tex [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+\textbf{\Dynawo-algorithms is a wrapper around  \href{https://dynawo.github.io/}{\underline{\Dynawo}} 
+that provides utility algorithms to calculate complex key values of a power system.} 
+
+It can be used with any tool of the \Dynawo toolsuite (DynaFlow, DynaWaltz, DynaSwing, DySym, DynaWave).\\
+
+It provides the following possibilities:
+\begin{itemize}
+  \item \textbf{Unitary simulations}: simulations of one or several power systems described in a \textbf{.jobs} file 
+  (see \href{https://github.com/dynawo/dynawo/releases/download/v1.3.0/DynawoDocumentation.pdf}{\underline{\Dynawo documentation}});
+  \item \textbf{Systematic analysis}: simulations of a same base power system subject to different N-k events to assess the global stability;
```
Comment (adriguir) [inline documentation/introduction/introduction.tex:32]: I would just stick to different events (not necessarily N-k)

**Review by adriguir — CHANGES_REQUESTED** [review]

**Review by adriguir — APPROVED** [review]

**Pull request merged**